custom/up_nrhm/data_sources/location_hierarchy.json

```diff
@@ -7,15 +7,31 @@
         "table_id": "location_hierarchy",
         "display_name": "Location Hierarchy",
         "configured_filter":  {
-             "operator": "eq",
-             "expression": {
-                 "datatype": null,
-                 "type": "property_name",
-                 "property_name": "is_active"
-             },
-             "type": "boolean_expression",
-             "comment": null,
-             "property_value": true
+            "type": "and",
+            "filters": [
+                {
+                    "operator": "eq",
+                    "expression": {
+                         "datatype": null,
+                         "type": "property_name",
+                         "property_name": "is_active"
+                    },
+                    "type": "boolean_expression",
+                    "comment": null,
+                    "property_value": true
+                },
+                {
+                    "operator": "eq",
+                    "expression": {
+                         "datatype": null,
+                         "type": "property_name",
+                         "property_name": "-deletion_date"
```

Comment:
Was it intentional to have the `-` at the start of this?

Reply:
@emord we have that ```-``` in the property name
![example_doc](https://user-images.githubusercontent.com/429541/52967735-40237300-33ab-11e9-9a89-52670828ae03.png)

Reply:
May be better to filter on `base_doc = "CouchUser-Deleted"` but this is fine .

Reply:
Using base_doc = "CouchUser-Deleted" is more in sync with how HQ checks if a user is deleted, so would be good to use that .

Reply:
@emord @mkangia updated

Reply:
thank you @lwyszomi